ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -0,0 +1,782 @@
+{
+  "comments": [
+    "NOTE: these tests only attempt to cover multi-join scenarios; ensuring the validity of the ",
+    "sub-join topologies is delegated to joins.json",
+    "",
+    "The first part of the name is the join sequence - STT, for example, means STREAM-TABLE-TABLE ",
+    "The second part of the name is the type of join - II, for example, means INNER INNER ",
+    "The third part of the name is descriptive of any additional information, if present."
+  ],
+  "tests": [
+    {
+      "name": "STT - II",
+      "statements": [
+        "CREATE STREAM S1 (ROWKEY INT KEY, ID bigint) WITH (kafka_topic='left', value_format='JSON');",
```

Comment:
Given you have `ksql.any.key.name.enabled` enabled... why are all your keys named `ROWKEY`?

Reply:
Suggest a better name?

Reply:
ID INT KEY, STRING NAME

Reply:
fair enough - i'll change this in my next PR!